cli/flags.rs

```diff
@@ -761,6 +762,7 @@ fn bundle_subcommand<'a, 'b>() -> App<'a, 'b> {
     )
     .arg(Arg::with_name("out_file").takes_value(true).required(false))
     .arg(ca_file_arg())
+    .arg(reload_arg())
     .arg(importmap_arg())
     .arg(unstable_arg())
```

Comment:
Please add a unit test similar to `bundle_with_lock` that tests `--reload` flag is accepted. Otherwise LGTM

Reply:
Beautiful! Thanks for pointing me in the right direction